app/views/layouts/application.html.erb

```diff
@@ -17,10 +15,8 @@
 
   <body>
     <%= render "shared/header" %>
-    <div id="su-wrap"><%# required to push the footer to the bottom of the page, rather than the viewport %>
-      <div id="su-content">
-        <%= yield %>
-      </div>
+    <div id="su-content">
```

Comment:
Did you find you needed the `su-content` div?

Reply:
Just tried without and works with the new css/footer, so will remove.

Reply:
Awesome!